modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/sign-in/index.js

```diff
@@ -16,20 +16,16 @@ const EMAIL_INPUT_ID = '_com_liferay_login_web_portlet_LoginPortlet_login';
 const PASSWORD_INPUT_ID =
 	'_com_liferay_login_web_portlet_LoginPortlet_password';
 
-const emailInput = document.getElementById(EMAIL_INPUT_ID);
-emailInput.setAttribute('placeholder', 'Email');
-emailInput.removeAttribute('value');
-
-const passwordInput = document.getElementById(PASSWORD_INPUT_ID);
-passwordInput.setAttribute('placeholder', 'Password');
+function inputValidation() {
+	const emailInput = document.getElementById(EMAIL_INPUT_ID);
+	const passwordInput = document.getElementById(PASSWORD_INPUT_ID);
 
-window.onload = function () {
-	document.getElementById(EMAIL_INPUT_ID).focus();
-};
+	window.onload = function () {
```

Comment:
Please, validate if this code has the same problema than Raylife create site

Reply:
Today this code doesn't mess with the rest of the application, but to prevent a future problem, I restructured this function.